alembic/env.py

```diff
@@ -0,0 +1,90 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+from app.config import DB_HOST, DB_PORT, DB_USER, DB_NAME, DB_PASS
+from app.database import Base
+
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+section = config.config_ini_section
+config.set_section_option(section, "DB_HOST", DB_HOST)
+config.set_section_option(section, "DB_PORT", DB_PORT)
+config.set_section_option(section, "DB_USER", DB_USER)
+config.set_section_option(section, "DB_NAME", DB_NAME)
+config.set_section_option(section, "DB_PASS", DB_PASS)
```

Comment:
will be unnecessary when you move everything to config file

Reply:
Could change to smth like this `config.set_main_option('sqlalchemy.url', <your config here>.db_url())`